### PR TITLE
Elastic Beanstalk deployment

### PR DESCRIPTION
**Pull request by allait**
Adds .ebextensions packages config, build script and updates gulp-sass

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [pep8]
-exclude = ./venv,./bower_components,./digital_marketplace_frontend_toolkit
```
Comment (robyoung) [inline setup.cfg]: Why does this also remove `./digital_marketplace_frontend_toolkit`? It doesn't mention it in the commit comment.

**Comment by TheDoubleK** [comment]
I think that the `digital_marketplace_frontend_toolkit` folder is no longer imported here at all.  It used to be brought in as part of the frontend build but is done in a different way now so we no longer need this exclusion from pep8.

**Comment by quis** [comment]
For reference, it's imported here now: https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/master/bower.json#L7

File: setup.cfg (hunk 1 above)
Comment (allait) [inline setup.cfg]: I've used the same file we have in other frontend apps. I couldn't find `digital_marketplace_frontend_toolkit` after running frontend build tasks, but as @quis mentioned it seems to be in `./bower_components/digital_marketplace_frontend_toolkit` now, which is ignored by `./bower_components`

**Pull request merged**